Makefile

```diff
@@ -0,0 +1,4 @@
+
+.PHONY: test
+test:
+	go test github.com/elastic/go-ucfg/...
```

Comment:
we did try to keep go-ucfg minimal. I'd vote for not having a make file and use the go tools instead. `go test ./...` or just `go test`.